src/main/webapp/app/exercises/text/assess/text-assessment.service.ts

```diff
@@ -267,4 +273,20 @@ export class TextAssessmentService {
                 .subscribe();
         }
     }
+
+    /**
```

Comment:
We should probably also extend the description and not only state params which we can also see in the method as well